src/GoogleBooks.php

```diff
@@ -19,7 +19,7 @@ class GoogleBooks
     /**
      * @var Client
      */
-    protected $http;
+    public $http;
```

Comment:
Or we could leave it protected and allow it to be injected in the constructor. Something like

```
public function __construct($options = [], Client $http = null)
    {
        $this->http = $http ?: new Client([
            'base_uri' => $this->baseUri,
            'handler' => isset($options['handler']) ? $options['handler'] : null,
        ]);
```

Reminds me that I would like to add http://httplug.io/, but let's keep that out for now :)

Reply:
hmm . I don't remember why I even made it public. It might have been a testing thing. I put it back to private and the methods and tests work. Do you know why I might have made it public? lol